From ff942caa0e94dcf9fdf3c3d2850729fe21cefedf Mon Sep 17 00:00:00 2001
From: David Hanina <dhanina@redhat.com>
Date: Mon, 17 Feb 2025 15:40:01 +0100
Subject: [PATCH 12/16] Mock read_text for acme unknown

---
 tests/test_meta.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/test_meta.py b/tests/test_meta.py
index 5cd9bb4..7ad0843 100644
--- a/tests/test_meta.py
+++ b/tests/test_meta.py
@@ -252,9 +252,11 @@ class TestMetaACME(BaseTest):
         assert result.kw.get('acme') == 'enabled'
 
     @patch('os.path.exists')
+    @patch('pathlib.Path.read_text')
     @patch('ipapython.ipautil.run')
-    def test_acme_unknown(self, mock_run, mock_exists):
+    def test_acme_unknown(self, mock_run, mock_result, mock_exists):
         mock_exists.return_value = True
+        mock_result.return_value = '1'
 
         mock_run.side_effect = [
             gen_result(
-- 
2.48.1

