From 4e021ab15d293e8eb0dd63e2f9a0812573a9e94a Mon Sep 17 00:00:00 2001
From: David Hanina <dhanina@redhat.com>
Date: Tue, 18 Feb 2025 09:57:10 +0100
Subject: [PATCH 14/16] Remove unnecessary timeout

---
 src/ipahealthcheck/meta/core.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/ipahealthcheck/meta/core.py b/src/ipahealthcheck/meta/core.py
index d9e4c49..56e7c20 100644
--- a/src/ipahealthcheck/meta/core.py
+++ b/src/ipahealthcheck/meta/core.py
@@ -36,10 +36,6 @@ class MetaCheck(Plugin):
                 proc_fips_enable_path = Path(paths.PROC_FIPS_ENABLED)
                 result_text = proc_fips_enable_path.read_text()
                 result = int(result_text)
-            except TimeoutError:
-                logger.debug('Reading %s timed out' % paths.PROC_FIPS_ENABLED)
-                fips = "check timed out"
-                rval = constants.ERROR
             except Exception as e:
                 logger.debug('Reading %s failed: %s' %
                              (paths.PROC_FIPS_ENABLED, e))
-- 
2.48.1

